spark/src/test/scala/org/apache/spark/sql/delta/skipping/ClusteredTableTestUtils.scala

```diff
@@ -60,6 +61,33 @@ trait ClusteredTableTestUtilsBase extends SparkFunSuite with SharedSparkSession
     assert(expectedClusteringColumns == actualClusteringColumns)
   }
 
+  // Verify the operation parameters of the last history event contains `clusterBy`.
+  protected def verifyDescribeHistoryOperationParameters(
+      table: String
+  ): Unit = {
+    val deltaLog = if (table.startsWith("tahoe.") || table.startsWith("delta.")) {
+      // Path based table.
+      DeltaLog.forTable(spark, table.drop(6).replace("`", ""))
+    } else {
+      DeltaLog.forTable(spark, TableIdentifier(table))
+    }
+    val clusteringColumns =
+      ClusteringColumnInfo.extractLogicalNames(deltaLog.unsafeVolatileSnapshot)
+    val lastEvent = deltaLog.history.getHistory(Some(1)).head
+    lastEvent.operation match {
+      case "CLUSTER BY" =>
+        // Operation is [[DeltaOperations.ClusterBy]] - ALTER TABLE CLUSTER BY
+        assert(
+          lastEvent.operationParameters("newClusteringColumns") === clusteringColumns.mkString(",")
```

Comment:
Does this still hold if you cluster by NONE?

Reply:
Yes, in that case, `clusteringColumns` will be an empty list.


We have a test that exercises this path: https://github.com/delta-io/delta/blob/49f2625423e16a7d0f6c7892145c7b3eed329052/spark/src/test/scala/org/apache/spark/sql/delta/skipping/clustering/ClusteredTableDDLSuite.scala#L563